Marketplace/src/main/java/com/example/Marketplace/controller/memberController.java

```diff
@@ -0,0 +1,41 @@
+//API를 이용하여 회원가입과 로그인 기능을 구현함.
+//이렇게 구현한다고 해서 바로 사용할 수는 없으므로, 실제로 로그인한사용자만 접근할수 있는 API를 정해주고, 로그인하지 않은 사용자도 접근할 수 있는 API를 지정해주고,, 접근제한, 통제 등을 구현해줘야함.
+//--> JWTA~~Filer클래스 구현하기
+
+package com.example.Marketplace.controller;
+
+import com.example.Marketplace.model.Auth;
+import com.example.Marketplace.security.TokenProvider;
+import com.example.Marketplace.service.MemberService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin(origins = "http://localhost:8080")
+@RequiredArgsConstructor
+@Transactional
+public class memberController {
```

Comment:
```suggestion
public class MemberController {
```
클래스 네이밍이 소문자로 시작하면 안됩니다. coding convention 에 대해 하나씩 살펴보시고 적용해보는 시도를 해보면 좋겠네요 ^^;

현업에서는 개발자들간의 같은 style 의 코드를 작성해보려는 노력을 하고 있습니다.
https://naver.github.io/hackday-conventions-java/